src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -67,30 +80,12 @@ class Config:
     sequences: Union[str, List[str]] = Field(
         description="The input sequences to generate the text from.",
     )
-    num_generated_predictions: int = Field(
```

Comment:
We are losing a lot of important descriptions of the arguments! Can we somehow preserve them for the users?

Reply:
they're all going to be moved to the GenerationConfig - why do we think we're using them?

Reply:
because GenerationConfig does not have any docstring or inline comments attached :(

Reply:
oh wait, confused with GenerationDefaults haha. Resolving.

Reply:
(is GenerationConfig coming in another PR?)

Reply:
ah so we have `generation_config` as an input argument, which lists the supported properties

Reply:
but its not implemented yet right?
Anyway, please rebase and then I will give green light